content/api/tracing/code_snippets/send_trace.py

```diff
@@ -1,29 +1,30 @@
-#
-#
-#
-#
-#
-#
-#
-#
-#
-#
-#
-#
-#
-# This is not yet supported by the python Client for Datadog API
-# Consult the curl example 
-#
-#
-#
-#
-#
-#
-#
-#
-#
-#
-#
-#
-#
-#
+import random
+import time 
+import requests
+
+# Create IDs.
+TRACE_ID=random.randint(1,1000000)
+SPAN_ID=random.randint(1,1000000)
+
+# Start a timer.
+START=int(time.time() * 1000000)
+
+# Do things...
+time.sleep(2)
+
+# Stop the timer.
+DURATION= int(time.time() * 1000000) - START
+
+# Send the traces.
+headers = {'Content-type': 'application/json'}
+data = {\
+	"trace_id": TRACE_ID,\
+	"span_id": SPAN_ID,\
+	"name": "span_name",\
+	"resource": "/home",\
+	"service": "service_name",\
+	"type": "web",\
+	"start": START,\
+	"duration": DURATION}
+	
+requests.put('http://localhost:8126/v0.3/traces', data = data, headers=headers)
```

Comment:
Let's ensure we have consistent python code styling with spaces around equals and lowercase var names e.g. `span_id = random...`

Reply:
creating a new card for this.